internal/command/postgres/config.go

```diff
@@ -30,7 +30,7 @@ func newConfig() (cmd *cobra.Command) {
 	cmd = command.New("config", short, long, nil)
 
 	cmd.AddCommand(
-		newConfigShow(),
```

Comment:
Was this intentional?

Reply:
No, good catch, edited the wrong config command by mistake. Thanks, should be fixed.